docs/index.md

```diff
@@ -9,13 +9,26 @@ This version of the installer, 3.12.0, installs Pulp 3.12.0 specifically.
 
 If run against an older version of Pulp 3, it will upgrade it to 3.12.0.
 
+---
+**Didn't find what you need to get started?**
+
+We are actively trying to make our documentation as complete and user-friendly as possible.
+If you experience any difficulties or have any feedback, write to `pulp-list@redhat.com`.
+[Documentation PRs](https://github.com/pulp/pulp_installer/edit/master/docs/index.md) are always welcome.
```

Comment:
I think we need something similar at the README, since the README goes to galaxy:
https://galaxy.ansible.com/pulp/pulp_installer